flang/lib/Lower/OpenMP/OpenMP.cpp

```diff
@@ -369,7 +369,9 @@ getDeclareTargetFunctionDevice(
 static llvm::SmallVector<const Fortran::semantics::Symbol *>
```

Comment:
Consider adding documentation for what this function does an the meaning of the variables.

Reply:
Done. I also simplified it a bit, since the return value wasn't necessary.